bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/AbstractConfiguration.java

```diff
@@ -525,6 +527,15 @@ public void setZkLedgersRootPath(String zkLedgersPath) {
         setProperty(ZK_LEDGERS_ROOT_PATH, zkLedgersPath);
     }
 
+    /**
+     * @return zk enable health path
+     * */
+    public String getEnableHealthPath() {
+        String zkLedgersRootPath = ZKMetadataDriverBase.resolveZkLedgersRootPath(this);
```

Comment:
we cannot refer to ZKMetadataDriverBase here, that is a base class.

We should move this method to the ZK implementation

Reply:
OK, I will move the getEnableHealthPath method implementation to the ZKMetadataDriverBase class.

Reply:
OK, I will move the getEnableHealthPath method implementation to the ZKMetadataDriverBase class.